### PR TITLE
Add Helm charts for installing plugin

### PR DESCRIPTION
**Pull request by kyoto**
Helm charts are taken from github.com/openshift/console-plugin-template

### CODE REVIEW

File: charts/openshift-console-plugin/values.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+plugin:
+  name: ""
```
Comment (jgbernalp) [inline charts/openshift-console-plugin/values.yaml:3]:
```suggestion
  name: "monitoring-plugin"
```

**Review by jgbernalp — COMMENTED** [review]
just a nit, the rest LGTM

File: charts/openshift-console-plugin/values.yaml (hunk 1 above)
Comment (kyoto) [reply charts/openshift-console-plugin/values.yaml]: 👍

**Pull request merged**